wheel_base: 0.095  # distance between wheels
wheel_radius: 3 # in cm
motor_max_speed: 100 # in RPM
target_time: 10 # seconds
i_state: 'i' 
f_state: 'f'
#i_state: [0.152, 0.213, 0.0,0.0]  # x,y,theta,velocity
#f_state: [1.31, 1.46, 2.0,0.0] 
c_state: 'i'
way_points: [[10,10,10,5,1],[5,3.5,2,2,2]] #updated by path planner
robot_radius: 0.07
start_navigation: True
job_done: False
t_left: 10.0
unload_time: 60 # in seconds
